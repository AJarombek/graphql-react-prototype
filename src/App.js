/**
 * Main application component for the GitHub dashboard.
 * @author Andrew Jarombek
 * @since 4/9/2020
 */

import { hot } from 'react-hot-loader/root';
import React from 'react';

const App = () => {
  return (
    <div className="app">
      <div className="header">
        <p>GraphQL React Prototype</p>
        <p>/</p>
        <p>GitHub</p>
        <p>/</p>
        <p>AJarombek</p>
      </div>

    </div>
  );
};

export default hot(App);
