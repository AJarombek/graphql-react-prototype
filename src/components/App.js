/**
 * Main application component for the GitHub dashboard.
 * @author Andrew Jarombek
 * @since 4/9/2020
 */

import { hot } from 'react-hot-loader/root';
import React from 'react';
import DashboardHeader from './DashboardHeader';
import DashboardItems from './DashboardItems';

const App = () => {
  return (
    <div className="app">
      <DashboardHeader />
      <DashboardItems />
    </div>
  );
};

export default hot(App);
