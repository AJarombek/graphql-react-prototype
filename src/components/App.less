/**
 * Stylesheet for the App component.
 * Author: Andrew Jarombek
 * Date: 4/9/2020
 */

.app {}
