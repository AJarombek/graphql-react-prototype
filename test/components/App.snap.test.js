/**
 * Snapshot test for the App component.
 * @author Andrew Jarombek
 * @since 4/25/2020
 */

import React from 'react';
import App from '../../src/components/App';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
