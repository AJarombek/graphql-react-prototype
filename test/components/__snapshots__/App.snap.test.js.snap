// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="app"
>
  <div
    className="dashboard-header"
  >
    <p>
      GraphQL React Prototype
    </p>
    <p>
      /
    </p>
    <p>
      GitHub
    </p>
    <p>
      /
    </p>
    <p>
      AJarombek
    </p>
  </div>
  <div
    className="dashboard-items"
  >
    <div
      className="items user-info"
    >
      <figure>
        <img
          alt=""
        />
      </figure>
      <h3 />
      <p />
      <a
        href="https:\\\\\\\\undefined"
      >
        Personal Website
      </a>
    </div>
    <div
      className="items repository-count"
    >
      <h3>
        Number of Repositories
      </h3>
      <h2>
        0
      </h2>
    </div>
    <div
      className="items top-language"
    >
      <p>
        Most Popular Programming Language
      </p>
      <h2>
        
      </h2>
      <p>
        Top language in
         
        <b>
          0
        </b>
         
        repositories.
      </p>
    </div>
    <div
      className="items most-popular-languages"
    >
      <h2>
        Most Popular Languages All-Time
      </h2>
    </div>
    <div
      className="items total-commits"
    >
      <h2>
        Most Total Commits
      </h2>
    </div>
    <div
      className="items trending-languages"
    >
      <h2>
        Trending Languages
      </h2>
    </div>
    <div
      className="items most-recent-commit"
    >
      <h2>
        Most Recently Committed
      </h2>
    </div>
    <div
      className="items most-productive-weeks"
    >
      <h2>
        Most Productive Weeks
      </h2>
    </div>
  </div>
</div>
`;
